tempesta_fw/cache.c

```diff
@@ -573,6 +626,8 @@ tfw_cache_add(TfwHttpResp *resp, TfwHttpReq *req)
 
 	if (!cache_cfg.cache || !tfw_cache_msg_cacheable(req))
 		return true;
+	if (!tfw_cache_employ(req, resp))
```

Comment:
`resp` isn't NULL here and you don't need `req` here. You only decide whether `resp` is cacheable here.